muhasin/index.js

```diff
@@ -0,0 +1,8 @@
+const colors = ["red", 9999, "green", "blue", 777, "yellow", "black", "purple", "brown"];
+
+
+
+
+
+const randomColor = colors[Math.floor(Math.random() * colors.length)];
+console.log("randomColor ==> ", randomColor);
```

Comment:
You still get numbers in console! After line 7 you have to filter numbers so you have to use if statement and the logic is, if you faced number from randomized variable which is randomColor you have to change it to an empty string! Try to find solution and if you had no clues just write me and we make a BO!